documentation/modules/upgrading/proc-upgrade-brokers-newer-kafka.adoc

```diff
@@ -110,14 +107,32 @@ If required, set the `version` property for Kafka Connect and MirrorMaker as the
 
 . _Optional step to reduce rolling updates_:
 +
-You can now perform one or more further product upgrades, starting with xref:proc-upgrading-the-co-{context}[]. 
+You can now perform one or more further product upgrades, starting with xref:proc-upgrading-the-co-{context}[].
 After reaching this step for the last upgrade you want to perform, you can complete this procedure.
 +
 Alternatively, skip this step and go to step 8.
 +
-. If the `log.message.format.version` and `inter.broker.protocol.version` of the _new_ Kafka version are _different from_ the old Kafka version, update the Kafka resource to use the new default versions. Otherwise, go to step 9.
+. If configured, update the Kafka resource to use the new `inter.broker.protocol.version` version. Otherwise, go to step 10.
```

Comment:
So, if `log.message.format.version` and `inter.broker.protocol.version` were _not_ configured (step 2), does the Cluster Operator update `Kafka.spec.kafka.config` with the new default versions?

Reply:
The versions are updated automatically